README.md

```diff
@@ -1055,6 +1055,7 @@ See also [Are we game yet?](http://arewegameyet.com)
   * [VictoremWinbringer/Victorem](https://github.com/VictoremWinbringer/Victorem) [[Victorem](https://crates.io/crates/Victorem)] — Easy UDP Game Server and UDP Client framework for creating simple 2D and 3D online game prototype [<img src="https://api.travis-ci.org/VictoremWinbringer/Victorem.svg?branch=master">](https://travis-ci.org/VictoremWinbringer/Victorem)
 * Voxlap
   * [bbodi/rust-voxlap](https://github.com/bbodi/rust-voxlap) — [Voxlap](http://advsys.net/ken/voxlap.htm) bindings
+* [Awesome wgpu-rs](https://github.com/rofrol/awesome-wgpu-rs) — A curated list of wgpu-rs code and resources
```

Comment:
This should be in the gaming header along with the "Games Made With Piston" section

Reply:
But it's not only for games. For example there are applications like code editor or pixel editor. https://github.com/rofrol/awesome-wgpu/blob/master/README.md#applications

Reply:
Good point, I didn't get that far :) The url seems to have changed since this was submitted, so can you correct that please?